### DISCUSSION TITLE
Display delayed trains as "active" in the "En route" area until they finally arrive at their destination in real time

### DISCUSSION THREAD

**Discussion by pascalwittler** [Traewelling/traewelling#2016] [Ideas / Feature Requests]
Current behaviour:

The train I checked in is currently about 40 minutes late. It was visible in the "En route" area (https://traewelling.de/statuses/active) until the time of the scheduled arrival at my destination, the progress bar was displayed correctly (something like 90%) and the real time next stop was also up to date.

Since the time of the scheduled arrival at my destination, my journey is no longer listed in the "En route" area, although the train is still on the way.

Requested new behaviour / new feature:

Display delayed trains as "active" in the "En route" section and on the map (same page) until they finally arrive at their destination in real time instead of just until the scheduled arrival.

Additional information and screenshots:

The details page of my journey still shows the correct status (not yet finished and therefore still active):

https://traewelling.de/status/194444

![image](https://user-images.githubusercontent.com/20669380/129639345-5621b796-b0f9-45ad-91d8-94ba102119c6.png)

**Comment by MrKrisKrisu** [comment]
@HerrLevin Does the beta version also look on the Scheduled time?

**Comment by HerrLevin** [comment]
As far as I remember the current behaviour might be the same as the beta.

**Comment by pascalwittler** [comment]
Since I noticed that the same behavior still applies, I take the opportunity to provide more explanatory screenshots:

[1] Status with real-time arrival time of the unfinished trip (https://traewelling.de/status/2744138)

![1](https://github.com/Traewelling/traewelling/assets/20669380/64ef0fd8-caec-49e3-9ef7-463fe342cb58)

[2] Overview map with „all active trips“ (https://traewelling.de/statuses/active around 22:30, after the scheduled arrival, before the real-time arrival)

![2](https://github.com/Traewelling/traewelling/assets/20669380/28ada6b7-269a-4300-86ba-50aa704681e0)
